historian/backend/src/main/java/com/hurence/logisland/historian/rest/v1/model/Selection.java

```diff
@@ -0,0 +1,210 @@
+        }
+
+    /**
+        * concatenation of owner with name for example
+    * @return id
+    **/
+        @JsonProperty("id")
+    @ApiModelProperty(required = true, value = "concatenation of owner with name for example")
+      @NotNull
+
+
+  public String getId() {
+    return id;
+    }
+
+        public void setId(String id) {
+        this.id = id;
+        }
+
+        public Selection name(String name) {
+        this.name = name;
+        return this;
+        }
+
+    /**
+        * Get name
+    * @return name
+    **/
+        @JsonProperty("name")
+    @ApiModelProperty(required = true, value = "")
+      @NotNull
+
+
+  public String getName() {
+    return name;
+    }
+
+        public void setName(String name) {
+        this.name = name;
+        }
+
+        public Selection owner(String owner) {
+        this.owner = owner;
+        return this;
+        }
+
+    /**
+        * Get owner
+    * @return owner
+    **/
+        @JsonProperty("owner")
+    @ApiModelProperty(required = true, value = "")
+      @NotNull
+
+
+  public String getOwner() {
+    return owner;
+    }
+
+        public void setOwner(String owner) {
+        this.owner = owner;
+        }
+
+        public Selection tags(List<String> tags) {
+        this.tags = tags;
+        return this;
+        }
+
+            public Selection addTagsItem(String tagsItem) {
+            this.tags.add(tagsItem);
+            return this;
+            }
+
+    /**
+        * list of tags id in selection
+    * @return tags
+    **/
+        @JsonProperty("tags")
+    @ApiModelProperty(required = true, value = "list of tags id in selection")
+      @NotNull
+
+
+  public List<String> getTags() {
+    return tags;
+    }
+
+        public void setTags(List<String> tags) {
+        this.tags = tags;
+        }
+
+        public Selection recordType(String recordType) {
+        this.recordType = recordType;
+        return this;
+        }
+
+    /**
+        * Get recordType
+    * @return recordType
+    **/
+        @JsonProperty("record_type")
+    @ApiModelProperty(required = true, value = "")
+      @NotNull
+
+
+  public String getRecordType() {
+    return recordType;
+    }
+
+        public void setRecordType(String recordType) {
+        this.recordType = recordType;
+        }
+
+
+    @Override
+    public boolean equals(java.lang.Object o) {
+    if (this == o) {
+    return true;
+    }
+    if (o == null || getClass() != o.getClass()) {
+    return false;
+    }
+        Selection selection = (Selection) o;
+        return Objects.equals(this.id, selection.id) &&
+        Objects.equals(this.name, selection.name) &&
+        Objects.equals(this.owner, selection.owner) &&
+        Objects.equals(this.tags, selection.tags) &&
+        Objects.equals(this.recordType, selection.recordType);
+    }
+
+    @Override
+    public int hashCode() {
+    return Objects.hash(id, name, owner, tags, recordType);
+    }
+
+
+@Override
+public String toString() {
```

Comment:
Indentation of this class is not great

Reply:
It is a class generated with swagger. Maybe we should modify the template ?